src/FirebirdSql.Data.FirebirdClient.Benchmarks/CommandBenchmark.cs

```diff
@@ -34,30 +34,46 @@ public Config()
 		{
 			var baseJob = Job.Default
 				.WithWarmupCount(3)
-				.WithToolchain(CsProjCoreToolchain.NetCoreApp60)
 				.WithPlatform(Platform.X64)
 				.WithJit(Jit.RyuJit);
+
+			AddJob(
+				baseJob
+					.WithToolchain(CsProjCoreToolchain.NetCoreApp80)
+					.WithCustomBuildConfiguration("ReleaseNuGet")
+					.WithId("NuGet80")
+					.AsBaseline()
+			);
+
+			AddJob(
+				baseJob
+					.WithToolchain(CsProjCoreToolchain.NetCoreApp80)
+					.WithCustomBuildConfiguration("Release")
+					.WithId("Core80")
+			);
+
 			AddDiagnoser(MemoryDiagnoser.Default);
-			AddJob(baseJob.WithCustomBuildConfiguration("Release").WithId("Project"));
-			AddJob(baseJob.WithCustomBuildConfiguration("ReleaseNuGet").WithId("NuGet").AsBaseline());
+
+			AddValidator(BaselineValidator.FailOnError);
+			AddValidator(JitOptimizationsValidator.FailOnError);
```

Comment:
If full JSON (*full.json) reports are exported then benchmarks results can be compared historically using [ResultsComparer](https://github.com/dotnet/performance/tree/main/src/tools/ResultsComparer). 

`AddExporter(JsonExporter.Full);`

Reply:
Never mind, I see how it is supposed to work now!